doc/Pacemaker_Explained/en-US/Ch-Utilization.txt

```diff
@@ -73,116 +73,118 @@ Then, specify the capacities the resources require:
 
 After you have configured the capacities your nodes provide and the
-capacities your resources require, you need to set the `placement-strategy`
+capacities your resources require, you need to set the +placement-strategy+
 in the global cluster options, otherwise the capacity configurations have
-`no effect`.
+'no effect'.
 
-Four values are available for the `placement-strategy`: 
+Four values are available for the +placement-strategy+: 
 
-`default`::
++default+::
 
-Utilization values are not taken into account at all, per default.
+Utilization values are not taken into account at all.
 Resources are allocated according to allocation scores. If scores are equal,
 resources are evenly distributed across nodes.
 
-`utilization`::
++utilization+::
 
-Utilization values are taken into account when deciding whether a node
-is considered eligible if it has sufficient free capacity to satisfy the
-resource's requirements. However, load-balancing is still done based on the
+Utilization values are taken into account 'only' when deciding whether a node
+is considered eligible (i.e. whether it has sufficient free capacity to satisfy
+the resource's requirements). Load-balancing is still done based on the
 number of resources allocated to a node. 
 
-`balanced`::
++balanced+::
 
 Utilization values are taken into account when deciding whether a node
-is eligible to serve a resource; an attempt is made to spread the resources
-evenly, optimizing resource performance.
+is eligible to serve a resource 'and' when load-balancing, so an attempt is
+made to spread the resources in a way that optimizes resource performance.
 
-`minimal`::
++minimal+::
 
-Utilization values are taken into account when deciding whether a node
-is eligible to serve a resource; an attempt is made to concentrate the
-resources on as few nodes as possible, thereby enabling possible power savings
-on the remaining nodes. 
+Utilization values are taken into account 'only' when deciding whether a node
+is eligible to serve a resource. For load-balancing, an attempt is made to
+concentrate the resources on as few nodes as possible, thereby enabling
+possible power savings on the remaining nodes. 
 
 
-Set `placement-strategy` with `crm_attribute`:
+Set +placement-strategy+ with `crm_attribute`:
 ----
-# crm_attribute --attr-name placement-strategy --attr-value balanced
+# crm_attribute --name placement-strategy --update balanced
 ----
 
 Now Pacemaker will ensure the load from your resources will be distributed
-evenly throughout the cluster - without the need for convoluted sets of
+evenly throughout the cluster, without the need for convoluted sets of
 colocation constraints.
 
-
 == Allocation Details ==
 
-=== Which node is preferred to be chosen to get consumed first on allocating resources? ===
+=== Which node is preferred to get consumed first when allocating resources? ===
 
-- The node that is most healthy (which has the highest node weight) gets
-consumed first.
```

Comment:
This is about the node-health-strategy and node-health-green/yellow/red, rather than resource scores though.